api-spec/wallet-api.yaml

```diff
@@ -1348,6 +1344,16 @@ components:
           type: string
       required:
         - paymentMethodType
+    CardBrand:
+      description: Payment instrument brand
+      type: string
+      enum:
+        - MASTERCARD
```

Comment:
Maybe mastercard can be removed since NPG returns MC for mastercard cards. What do you think?

Reply:
Yes I'm agree. Also in logo mapping should it be removed? Or leave it as a possible alias? (considering future refactor of brand as a string)